ui/src/views/mint-site/mint-site.tsx

```diff
@@ -65,7 +65,7 @@ export const MintSite = () => {
         await FleekERC721.mint(
           {
             name,
-            description,
+            description: description.replaceAll(/\n/g, '\\n'), //replace break lines with \\n so it doesn't break the json
```

Comment:
Either you send with extra `\` or add it when receiving the query response, doing both seems inaccurate. I would send the text as it is and parse it when reading, because the text will be stored as it is. For other client sides that may be different from JS, I think it will be correct.

But thinking about the Opensea it will require to be correct when sending because they are not parsing it when they read, so probably the best approach would be remove the parsing on `ui/src/integrations/ethereum/lib/fleek-erc721.ts`.

Anyways, I have tested:
```js
'test\\ntestetes\\n'.replaceAll(/\n/g, '\\n')
```
which produces:
```
'test\\ntestetes\\n'
```
So keeping both parses when writing and when reading is also fine.